.openpublishing.redirection.json

```
@@ -3862,7 +3862,7 @@
     },
     {
       "source_path": "docs/visual-basic/sample-applications.md",
-      "redirect_url": "https://github.com/dotnet/samples/tree/master/snippets/visualbasic"
+      "redirect_url": "/samples/browse/?languages=vb"
```

```suggestion
      "redirect_url": "https://docs.microsoft.com/samples/browse/?languages=vb"
```

I think this is a good place. We can use the full URL since the samples browser isn't published in an offline format.